### PR TITLE
scenarios: Test that the apex service is removed in traffic split

### PR DESCRIPTION
**Pull request by draychev**
This test passing shows that the APEX services is no longer counted when TrafficSplit is present.

Ref https://github.com/openservicemesh/osm/issues/1979

---

**Affected area**:

- New Functionality      [ ]
- Documentation          [ ]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

### CODE REVIEW

**Comment by ksubrmnn** [comment]
Can you link to the pull request that fixed this issue? I'm wondering why this test wasn't also changed with that fix. 

I didn't see the PR associated with the issue you referenced.

File: tests/scenarios/traffic_split_with_apex_service_test.go [hunk 1]
```diff
@@ -151,11 +144,11 @@ var _ = Describe(``+
 
 				expectedWeightedCluster := &xds_route.WeightedCluster{
 					Clusters: []*xds_route.WeightedCluster_ClusterWeight{
-						weightedCluster("bookstore-apex", 100),
+						// weightedCluster("bookstore-apex", 100),
```
Comment (ksubrmnn) [inline tests/scenarios/traffic_split_with_apex_service_test.go:147]: Any reason why we can't just remove this?

**Review by ksubrmnn — APPROVED** [review]

**Review by snehachhabria — APPROVED** [review]

File: tests/scenarios/traffic_split_with_apex_service_test.go (hunk 1 above)
Comment (draychev) [reply tests/scenarios/traffic_split_with_apex_service_test.go:147]: I felt compelled to leave it here to explicitly show that the apex cluster has been removed -- a documentation of a kind.

**Comment by draychev** [comment]
> Can you link to the pull request that fixed this issue? I'm wondering why this test wasn't also changed with that fix.

@ksubrmnn  These tests did not run automatically in CI. I'm going to change this with the following PR:  https://github.com/openservicemesh/osm/pull/2303

I believe this issue was fixed with the following PR https://github.com/openservicemesh/osm/pull/2110 by @addozhang

**Pull request merged**